test/everypolitician/popolo/membership_test.rb

```diff
@@ -51,4 +51,9 @@ def test_membership_end_date_method_always_present
   def test_membership_person
     assert_equal 'Bob', popolo.memberships.first.person.name
   end
+
+  def test_membership_equality
```

Comment:
This is slightly confusingly named as this is testing membership _in_-equality. It's primarily an issue because I think you should have an explicit test that checks equality too.